docs/table_design/Data_distribution.md

```diff
@@ -142,27 +148,75 @@ CREATE TABLE site_access
+
+  - Method 2: manually set the number of buckets
+
+    If you intend to set the number of buckets, StarRocks 2.4 and later versions support using multiple threads to scan a tablet in parallel during a query, thereby reducing the dependency of scanning performance on the tablet count. We recommend that each tablet contains about 10 GB of raw data. You can estimate the amount of data in each partition of a table and then decide the number of tablets. To enable the parallel scanning on tablets, make sure the `GLOBAL enable_tablet_internal_parallel` is enabled.
+
+    ```sql
+    CREATE TABLE site_access (
+        site_id INT DEFAULT '10',
+        city_code SMALLINT,
+        user_name VARCHAR(32) DEFAULT '',
+        pv BIGINT SUM DEFAULT '0')
+    AGGREGATE KEY(site_id, city_code, user_name)
+    DISTRIBUTED BY HASH(site_id,city_code) BUCKETS 30; -- Suppose the volume of raw data that loads into a partition is 300 GB. Because we recommend that each tablet contains 10 GB of raw data, the number of buckets can be set to 30.
+    ```
+
+- How to set the number of buckets when a partition is created
 
-If you intend to set the number of buckets, StarRocks 2.4 and later versions support using multiple threads to scan a tablet in parallel during a query, thereby reducing the dependency of scanning performance on the tablet count. We recommend that each tablet contains about 10 GB of raw data. You can estimate the amount of data in each partition of a table and then decide the number of tablets. To enable the parallel scanning on tablets, make sure the `GLOBAL enable_tablet_internal_parallel` is enabled.
+  - Method 1: automatically set the number of buckets (Recommended)
 
-> Note: You cannot modify the number of tablets for an existing partition. You can only modify the number of tablets when you add a partition.
+    Since v2.5.7, StarRocks supports automatically setting the number of buckets based on machine resources and data volume for a partition.
+    To enable this feature, make sure that the FE dynamic parameter `enable_auto_tablet_distribution` remains the default value `true`.
+
+    To disable this feature, run the `ADMIN SET FRONTEND CONFIG ('enable_auto_tablet_distribution' = 'false');` statement. And when a new partition is added without specifying the number of buckets, the new partition inherits the the number of buckets set when a table is created. After a new partition is added successfully, you can execute [SHOW PARTITIONS](../sql-reference/sql-statements/data-manipulation/SHOW%20PARTITIONS.md) to view the number of buckets automatically set by StarRocks for the new partition.
+
+  - Method 2: manually set the number of buckets
+
+    You can also manually specify the bucket count when adding a new partition. To calculate the number of buckets for a new partition, you can refer to the approach used when manually set the number of buckets when a table is created, as mentioned above.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/48719753-1273-4f6b-8f27-d47fcd1eb391)
-> used when manually setting the number of buckets at table creation